lib/unobtainium/world.rb

```diff
@@ -163,14 +164,26 @@ def resolve_options(label, options)
         options.delete("base")
       end
 
+      # we really need :caps and "desired_capabilities" in our options
+      unless options.has_key?(:caps) # rubocop:disable Style/PreferredHashMethods
```

Comment:
I think, `Hash#key?` and `Hash#has_key?` behave the same. Is that right? If so, I'd rather use one of the methods in all places.